strapi/strapi
# [Custom Fields] Handle empty state layout

### What does it do?

- Adds the empty state layout to the custom fields tabs
- Updates the tests

### Why is it needed?

- Guide users to the marketplace when no custom fields are found

### How to test it?

- Go to any content-type in the content-type builder
- Click the "add attribute" button
- Click the "Custom" tab,  you should see the empty layout
